WoWPro_Leveling/Wrath/Neutral/WOTLK_ZulDrak.lua

```diff
@@ -68,62 +68,62 @@ C Dressing Down|QID|12648|N|Return to the Reliquary of Agony and use the choker.
+C Feedin' Da Goolz|QID|12652|N|Use the bowl near the ghouls. Be careful of the Blightguards, they see through your disguise. You should hotkey the chocker because your diguise will fail at some point.|U|38701|M|21.24,75.01|
 T Feedin' Da Goolz|QID|12652|N|Back to Gristlegut.|M|19.9,75.5|U|38699|
 T Dressing Down|QID|12648|O|N|Back to Ebon Watch.|M|14.12,73.79|
-T Suit Up!|QID|12649|O|N|Back to Ebon Watch.|M|14.12,73.79|
+T Suit Up!|QID|12649|O|N|Back to Stefan Vadu.|M|14.12,73.79|
 A Infiltrating Voltarus|QID|12661|PRE|12648^12649|M|14.1,73.8|N|From Stefan Vadu.|
 T Gymer's Salvation|QID|12914|N|Back to Crusader Forward Camp.|M|25.34,63.97|
 A Our Only Hope|QID|12916|PRE|12914|M|25.3,64.0|N|From Engineer Reed.|
 C Our Only Hope|QID|12916|N|Use the Jumbo Seaforium Charge on the cage.|M|19.65,56.47|
 T Our Only Hope|QID|12916|M|19.7,56.4|N|To Gymer.|
-A The Storm King's Vengeance|QID|12919|PRE|12916|N|From Gymer.|
-N The Storm King's Vengeance|QID|12919|N|Do not bother to look for the scourge: they will find you.|
-N The Storm King's Vengeance|QID|12919|N|Talk to Gymer to mount him and go!|
-C The Storm King's Vengeance|QID|12919|QO|1|N|Finish killing scourge.|S|
+A The Storm King's Vengeance|QID|12919|PRE|12916|N|From Gymer.|M|19.7,56.4|
+C The Storm King's Vengeance|QID|12919|N|Talk to Gymer to mount him and go! (Option 3)|M|19.7,56.4|S!US|QO|1|
 K Algar the Chosen|QID|12919|QO|2|M|29.2,46|N|He flies in circles underneath Volatrus, above the circular platform on the ground.|
 K Prince Navarius|QID|12919|QO|4|M|29.8,65.4;32.4,63.6|CN|N|At one of these two spots.|
-K Thrym|QID|12919|QO|3|N|He will spawn after killing the Prince. Snack on clouds and wait.|
-C The Storm King's Vengeance|QID|12919|QO|1|N|Finish killing scourge.|US|
+K Thrym|QID|12919|QO|3|N|He will spawn after killing the Prince. Snack on clouds and wait.|M|PLAYER|
 T The Storm King's Vengeance|QID|12919|N|Back to Crusader Forward Camp.|M|25.43,63.97|
-A Dark Horizon|QID|12664|O|N|You got this because you didn't finish the Drakuru quest chain in Grizzly Hills.|M|27.1,46.2|
-C Dark Horizon|QID|12664|O|N|Circle around the platform until you find a ramp on the outer ring. Gorebag is up there. Enjoy the ride.|M|29.76,47.90|
-T Dark Horizon|QID|12664|O|M|27.1,46.2|N|To Overlord Drakuru.|
-A Reunited|QID|12663|O|N|You got this because you finished the Drakuru quest chain in Grizzly Hills.|M|27.1,46.2|
-C Reunited|QID|12663|O|N|Circle around the platform until you find a ramp on the outer ring. Gorebag is up there. Enjoy the ride.|M|29.76,47.90|
-T Reunited|QID|12663|O|N|To Overlord Drakuru.|M|27.1,46.2|
-T Infiltrating Voltarus|QID|12661|N|Port back down, blow the horn.|U|41390|
+R Volatrus|ACTIVE|12661|M|28.16,44.84|N|Use the Choker and step onto the center of the pedestal|U|38699|CC|
+A Dark Horizon|PRE|-12238|QID|12664|N|You got this because you didn't finish the Drakuru quest chain in Grizzly Hills.|M|27.1,46.2|
```

Comment:
Seems quest ID 12238 is the one you need up at the top for the Drakuru quest chain PRE tag.